app/controllers/Api.scala

```diff
@@ -169,6 +169,15 @@ trait Api extends Logging {
     singleItem(Prism.instanceAgent, arn)
   }
 
+  // for some bizarre compilation reason this can't be called 'lambdaList'
```

Comment:
If we think `function` isn't intention revealing enough, would `getAllLambdas` work?

Reply:
done https://github.com/guardian/prism/pull/71/commits/1de2c079bd88d8ef0134c6811e832f65d0313af3

Reply:
retronym (scala contributor) [advises](https://github.com/wix/accord/issues/84#issuecomment-262652711)

> In general, I wouldn't recommend using the experimental GenBCode backend in 2.11.x as there are known bugs which only fixed in 2.12.x.

hence try removing compiler flag

```
-Ybackend:GenBCode
```

Reply:
@mario-galic thank you for your investigation work that got to the bottom of it - one of the weirdest bugs I've ever seen
I've implemented your suggestion which allowed me to make the new function names consistent with all the others - see https://github.com/guardian/prism/pull/71/commits/46fed7f1c2e8968d5ae4e21b132b8dbf638745d9
Thanks